.circleci/config.yml

```diff
@@ -3,86 +3,79 @@
 version: 2.1
+orbs:
+  python: circleci/python@<< pipeline.parameters.python-orb-version >>
+  node: circleci/node@<< pipeline.parameters.node-orb-version >>
+  orb-tools: circleci/orb-tools@10.0
+
+parameters:
+  python-orb-version:
+    type: string
+    default: "2.0.3"
+  node-orb-version:
+    type: string
+    default: "5.0.1"
+  python-cache-version:
+    type: string
+    default: v3
+  dev-cache-version:
+    type: string
+    default: v3
+  node-cache-version:
+    type: string
+    default: v3
+  global-docker-image:
+    type: string
+    default: cimg/python:3.9.10-node
+
+
 references:
     tag_filter: &tag_filter
       filters:
         tags:
           only: /^v\d+\.\d+\.\d+$/
 
-    reqs_tox: &reqs_tox
-      requires:
-        - prepare-tox3-7
-
     reqs_content_checkout: &reqs_content_checkout
       requires:
         - checkout-content
-        - prepare-tox3-7
-
-    create_requirement_files: &create_requirement_files
-      run:
-        name: Create requirement files
-        command: |
-          pip install pipfile-requirements
-          pipfile2req > requirements.txt
-          pipfile2req -d > requirements-dev.txt
 
-    restore_tox_37_cache: &restore_tox_37_cache
-      restore_cache:
-        key: tox-37-cache-{{ checksum "Pipfile" }}-{{ checksum "Pipfile.lock" }}
-
-    save_tox_37_cache: &save_tox_37_cache
-      save_cache:
-        key: tox-37-cache-{{ checksum "Pipfile" }}-{{ checksum "Pipfile.lock" }}
-        paths:
-          - .tox/py37
+    install_node_ci: &install_node_ci
+      - node/install-packages:
+          cache-version: << pipeline.parameters.node-cache-version >>
+          
+    install_demisto_sdk: &install_demisto_sdk
+      - python/install-packages:
+          pkg-manager: "pipenv"
+          args: "--system"
+          cache-version: << pipeline.parameters.python-cache-version >>
+          pre-install-steps:
+            - run:
+                name: export demisto-sdk to path
+                command: echo 'export PYTHONPATH="${PYTHONPATH}:${HOME}/project"' >> $BASH_ENV
+      - <<: *install_node_ci
 
 jobs:
-  tox3-7-unit-tests:
-      # docker image circleci/python:3.8.3-buster-node also contains python 3.7
+  pytest:
+      parameters:
+        pythonversion:
+          type: string
       docker:
-        - image: circleci/python:3.8.3-buster-node
+        - image: << parameters.pythonversion >>
       steps:
         - checkout
-        - *create_requirement_files
-        - *restore_tox_37_cache
-        - run:
-            # install npm modules so readme tests run with mdx verification
-            name: npm install
-            command: |
-              npm ci
-        - run:
-            name: Setup Tox
-            command: |
-              pip install tox
+        - python/install-packages:
+            pkg-manager: "pipenv"
+            args: "--dev --system"
+            cache-version: << pipeline.parameters.python-cache-version >>
```

Comment:
I don't understand why the cache version is a parameter when it's value is `v3` - if it's relatively constant such as a major version then it doesn't need to be a parameter - and if it isn't, then it's value probably should look other than something like `v3`.